cds_ils/cli.py

```diff
@@ -223,6 +237,44 @@ def run_command(command, catch_exceptions=False, verbose=True):
     run_command("vocabulary index opendefinition opendefinition --force")
 
 
+def recreate_cover(pid):
+    # search doc
+    record = Document.get_record_by_pid(pid)
+    indexer = DocumentIndexer()
+    if not record:
+        record = Series.get_record_by_pid(pid)
+        indexer = SeriesIndexer()
+    if record:
+        pick_identifier_with_cover_task(record)
+        indexer.index(record)
+        click.secho("Recreated cover for PID: {}".format(pid), fg='blue')
+    else:
+        click.secho("No document or series found with PID: {}", fg='red')
+
+
+@covers.command()
+@click.argument("pid", nargs=1, type=str)
+@with_appcontext
+def recreate(pid):
+    recreate_cover(pid)
+
+
+@covers.command()
+@with_appcontext
+def recreate_all_documents():
+    all_documents = DocumentSearch().scan()
```

Comment:
```suggestion
    all_documents = current_app_ils.document_search_cls().scan()
```
same for series, to avoid hardcoding

Reply:
actually, would it be "safer" if we query the DB?
Get from PID store all the `docids` that are registered, in batches, and iterate. WDYT?

Reply:
You might have missed my comment maybe... WDYT?